### DISCUSSION TITLE
Line Wrap for Copilot Ghost Text in Neovim

### DISCUSSION THREAD

**Discussion by joswr1ght** [community/community#126244] [Copilot Conversations]
### Select Topic Area

Question

### Body

When working with Copilot and Neovim, I find that the suggested ghost text often disappears off the edge of the terminal:

<img width="950" alt="image" src="https://github.com/community/community/assets/1318817/839f0bf9-7a4f-4163-9972-0b43c73ef681">

The ghost text from the Copilot + Neovim plugin does not honor the `set wrap` soft attribute, making it difficult to use the suggestions from Copilot.

Is it possible to get soft wrap support for Copilot ghost text suggestions?

**Comment by joswr1ght** [comment]
Following up on this request. Copilot's usefulness is limited when the author can't see what Copilot is suggesting. Thank you.

**Comment by joswr1ght** [comment]
Following up on this request. Thank you!

**Reply by styopazharkov** [reply]
have you managed to find a solution for this? I am having the same issue.

**Reply by joswr1ght** [reply]
Sadly, no response from the Copilot team.
